addons/onboarding/controllers/onboarding.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import http
+from odoo.http import request
+
+
+class OnboardingController(http.Controller):
+    @http.route(['/onboarding/<int:onboarding_id>',
+                 '/onboarding/<string:onboarding_alias>'], auth='user', type='json')
```

Comment:
alias -> banner_route ?

Reply:
Idea was to get rid of banner_route and use only the ID, to match what we usually do for routes, but it seems current implementation does not allow it.
So yes, we can go back to only banner_route... sorry @flch-odoo 😬

Reply:
went back to `route_name` because the existing `banner_route` attributes will we set to `/onboarding/{route_name}`.